internal/driver/driver.go

```diff
@@ -42,7 +42,7 @@ func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
 	d.sdk = sdk
 
 	// Define custom API endpoints
-	if err := d.sdk.AddRoute("/api/v3/call", handleMethodCall, http.MethodPost); err != nil {
+	if err := d.sdk.AddCustomRoute("/api/v3/call", interfaces.Authenticated, handleMethodCall, http.MethodPost); err != nil {
```

Comment:
Is this the internal EdgeX authentication method?

Reply:
Yes. The authentication check occurs internally when the method is called.